src/eon.erl

```diff
@@ -5,6 +5,7 @@
 
 %%%_* Module declaration ===============================================
 -module(eon).
+-feature(maybe_expr, disable).
```

Comment:
Why is this feature disabled?

Reply:
This library depends on `stdlib2` that exposes a type called `maybe` that clashes with the feature being enable in Erlang, by default. We disable it to move forward without having to deal with all the dependencies (especially since it's likely `stdlib2` will leave in the future).

Reply:
Lemme see if using the atom is better/possible.

Reply:
No, I am with you here, even if we should remove stdlib2 as quickly as possible it requires lots of rewrites.

Reply:
I pushed an alternative approach. Not sure we'll need `maybe` in the future, here, but best leave that option open. Thanks.

Reply:
Was that all that need, OK, cool!